test/initializeConnection.testSuite.ts

```diff
@@ -6,10 +6,6 @@ import { WalletProvider } from "../src/types/WalletProvider";
 const iam_withoutKey = new IAM({ rpcUrl });
 
 export const initializeConnectionTests = () => {
-    test("initializeConnection requires privateKey or walletProvider enum", async () => {
```

Comment:
Why do you think this test is not needed anymore?

Reply:
Because in node env in constructor we are setting providerType always. So it is not possible to have not defined walletProvider.
I was trying to use spyOn this property, but without success.

Reply:
But the test anyway should confirm that error will be thrown when private key is not provided, just with different error message

Reply:
Ok, i changed error message there.